bitgouel-batch/src/main/kotlin/team/msg/common/BaseJobParameter.kt

```diff
@@ -0,0 +1,8 @@
+package team.msg.common
+
+/**
+ * 개발/QA 도중에 같은 Job 파라미터 값으로 배치 실행이 가능하도록 version을 지정합니다.
+ * version은 1부터 시작합니다.
+ */
+open class BaseJobParameter(val version: Int) {
+}
```

Comment:
```suggestion
open class BaseJobParameter(val version: Int)
```
클래스에 생성자 매개변수만 있으니까, 괄호를 생략하는게 더 보기좋은 것 같아요! 어떻게 생각하시나요?

Reply:
넵 반영하겠습니다.